gallery/src/styles/NxBtn/NxBtnPage.tsx

```diff
@@ -62,6 +63,13 @@ const NxBtnPage = () =>
               <NxTable.Cell>Modifier on <NxCode>.nx-btn</NxCode></NxTable.Cell>
               <NxTable.Cell>Makes the button a "tertiary" button</NxTable.Cell>
             </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>nx-btn--small</NxCode></NxTable.Cell>
+              <NxTable.Cell>Modifier on <NxCode>.nx-btn</NxCode></NxTable.Cell>
+              <NxTable.Cell>
+                Buttons that typically are used in situations where patterns that contain a button are repeated
```

Comment:
I find this description confusing.  I see you copied it from the mockup, but left off part of the description which, in my opinion, made it a lot clearer.  Consider restoring that extra text.

Reply:
https://github.com/sonatype/sonatype-react-shared-components/pull/1220/commits/a368678059041651ca20de36e398d78bd968e5c9